gbeaudry321/ser401-NASA-psyche-mission
# Us327 psyche asteroid ip gbeaudry

Info Point Documentation merge

I cleaned up a bunch of files with unused stuff that resulted from some of the tutorials I used in working through the info point template. Hence all the edited files listed.

Most importantly I added the Docs/Architecture/InfoPoints/info-point.md 
This both describes the info points and how to create/edit info points. A work in progress, but wanted basics for others to use if necessary.

To Test:
- open info-point.md
- Check if you can follow the part at the bottom under Editing and Maintaining within the Program

Expected behavior:
- if you used it to create a new info point, then there should be a new interactive info point on the map.
- if you just followed the instructions without implementing, there should be no behavior.
